nautobot/core/api/views.py

```diff
@@ -197,6 +211,12 @@ def initial(self, request, *args, **kwargs):
         """
         super().initial(request, *args, **kwargs)
 
+        # Django Rest Framework stores the raw API version string e.g. "1.2" as request.version.
+        # For convenience we split it out into integer major/minor versions as well.
+        major, minor = request.version.split(".")
```

Comment:
What is the result of this if `1.2.3` is sent in as the API version for example?